roles/docker/tasks/cleanup.yml

```diff
@@ -12,3 +12,9 @@
   become: true
   with_items:
     - "{{ docker_api_port }}"
+
+- name: cleanup contiv-compose
```

Comment:
Yes, obviously it will force a download of contiv-compose iff the user initiates a restart with the -r option. Given the priorities, I don't have the bandwidth to add versioning for compose. Meanwhile, from a user's perspective, this is broken. I think we should fix what is broken and then work on making it perfect. Alternatively, if you can submit a fix for this, I am fine with it.